### ISSUE TITLE
Dependencies broken

### ISSUE THREAD

**Issue by mzabaluev** [libp2p/rust-libp2p#1654]
Steps to repro:

1. Check out `master`, `v0.21.1` or unpack the release tarball.
2. Run `cargo update`.
3. Build the crate.

The build fails with
> error[E0277]: the trait bound `async_tls::connector::TlsConnector: std::convert::From<std::sync::Arc<rustls::client::ClientConfig>>` is not satisfied
  --> transports/websocket/src/tls.rs:73:47
   |
73 |             client: Arc::new(client_config()).into(),
   |                                               ^^^^ the trait `std::convert::From<std::sync::Arc<rustls::client::ClientConfig>>` is not implemented for `async_tls::connector::TlsConnector`
   |
   = help: the following implementations were found:
             <async_tls::connector::TlsConnector as std::convert::From<std::sync::Arc<rustls::client::ClientConfig>>>
   = note: required because of the requirements on the impl of `std::convert::Into<async_tls::connector::TlsConnector>` for `std::sync::Arc<rustls::client::ClientConfig>`

**Comment by mzabaluev** [comment]
I'd wager it has to do with this:

```
$ cargo tree
...
├── libp2p-websocket v0.21.0 (/home/mzabaluev/work/iohk/rust-libp2p/transports/websocket)
│   ├── async-tls v0.7.1
│   │   ├── rustls v0.18.0
...
│   ├── rustls v0.17.0
```

**Comment by fangzhongwei** [comment]
default-features = false

**Comment by yoshuawuyts** [comment]
Yanked `async-tls` `v0.7.1`, re-published as `v0.8.0`. That should resolve this issue I believe.

**Comment by mzabaluev** [comment]
> default-features = false

Anything that uses the "websockets" feature is affected. But no worries any more, thanks to @yoshuawuyts. I will update #1657 to not require the yanked release.
